test/160_claim_3_test.sh

```diff
@@ -32,10 +32,11 @@ stop_router_on $HOST1
 stop_router_on $HOST2
 
 # Now make host1 attempt to claim from host2, when host2 is stopped
+# checking whether host1 will hang trying to talk to host2
 weave_on $HOST2 launch-router
 # Introduce host3 to remember the IPAM CRDT when we stop host2
 weave_on $HOST3 launch-router $HOST2
 stop_router_on $HOST2
-weave_on $HOST1 launch-router $HOST3
+assert_raises "DOCKER_HOST=tcp://$HOST1:$DOCKER_PORT timeout 30 $WEAVE launch-router $HOST3"
```

Comment:
Hmm. I'm not sure this is worthwhile. I'd leave the code alone and only add the explanatory comment.